### DISCUSSION TITLE
Server down?

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#183] [General]
Hallo

bei uns melden mehrere Praxen das das generieren der Impfzertifikate seit heute morgen nicht mehr geht.

**Comment by elpatron68** [comment]
Das können wir bestätigen. 
![image](https://user-images.githubusercontent.com/4498667/130192614-4ac8c539-4d3b-448a-92e1-4547b7289666.png)

**Reply by oh-t2med** [reply]
Wir können das auch bestätigen. Es sieht also nach einer zentralen Fehlfunktion aus.

**Reply by elpatron68** [reply]
Erste Kunden berichten, daß es anscheinend wieder funktioniert. /medisoftware

**Reply by Faldebbagh** [reply]
seit zwei tagen geht wieder nicht . 

HTTP/1.1 400 Bad Request
..
<div id="kc-error-message">
    <p class="instruction">An error occurred, please login again through your application.</p>
</div>
...

**Reply by thinkberg** [reply]
Ist das ueber eine Browseranwendung oder PVS?
Wenn Browser, welcher Browser ist das genau?

Danke.

**Reply by Faldebbagh** [reply]
das  war PVS 
ab heute morgen habe die Praxen nochmal gemeldet , es geht wieder .

**Reply by thinkberg** [reply]
Alles klar, super.
